src/test/java/org/opentripplanner/transit/raptor/_data/transit/TestAccessEgress.java

```diff
@@ -23,6 +24,7 @@ public class TestAccessEgress implements RaptorAccessEgress {
   private final int cost;
   private final int numberOfRides;
   private final boolean stopReachedOnBoard;
+  private final boolean isZeroDuration;
```

Comment:
Should we rename this to empty, and add a method overriding `isEmpty`?

Reply:
yeah i renamed it as well - and added the override method.